lib/slack/web/client.rb

```diff
@@ -6,7 +6,7 @@ class Client
       include Faraday::Request
       include Api::Endpoints
 
-      attr_accessor(*Config::ATTRIBUTES)
+      attr_accessor(*Config::ATTRIBUTES, :oauth_scopes)
```

Comment:
I thought about making this `attr_reader :oauth_scopes` so that it wouldn't appear that you can set the scopes here. But then it would need to use `instance_variable_set` from the middleware to set them. Not sure which is better.